profiler/profiler_test.go

```diff
@@ -36,7 +36,6 @@ func TestStart(t *testing.T) {
 		if host, err := os.Hostname(); err != nil {
 			assert.Equal(host, activeProfiler.cfg.hostname)
 		}
-		assert.Equal(defaultAPIURL, activeProfiler.cfg.apiURL)
```

Comment:
Why did you remove this? Accident?

Reply:
Not sure how i managed to make that mistake...